sympy/printing/latex.py

```diff
@@ -2482,6 +2482,9 @@ def _print_Feedback(self, expr):
         denom = self._print(den)
         return r"\frac{%s}{%s}" % (numer, denom)
 
+    def _print_TransferFunctionMatrix(self, expr):
+        return self._print(expr._expr_mat)
```

Comment:
Should this not look different from an ordinary matrix in some way? It can be very confusing when different objects print the same but behave differently.

Reply:
It is represented like a typical matrix in every academic material I have read related to it, even in control packages in other modules. If you have any suggestions then maybe we can discuss them.

Reply:
If it is a Matrix then we don't need a separate class. Apparently it needs to be different from a matrix so it should probably look a little different. For example in Mathematica it seems to display with a funny `tau` next to it.

Reply:
It makes sense, but what can be the best way to add this `tau` here? 
The `tau` in Mathematica is reminiscent of `TransferFunction`, and similarly, they have curly `S` for State Space representation. A comparable argument should be valid for `TransferFunction` and `Expr`. Should we also consider adding `tau` to `TransferFunction`?

Reply:
@namannimmo10 @oscarbenjamin, should I go ahead with adding `tau` as a subscript?

![image](https://user-images.githubusercontent.com/53227127/123742290-0773fc80-d8c9-11eb-83ef-034fa31701c6.png)

After it is implemented, `TransferFunctionMatrix` would render like this.

Reply:
We can add `tau` as a superscript for it to look different from an ordinary matrix.

Reply:
> We can add `tau` as a superscript for it to look different from an ordinary matrix.

Superscript might confuse users with exponent. Subscript is more apt, in my opinion.

Reply:
Also, should I change only the latex method or both latex and pprint?

Reply:
Both of them.

Reply:
@namannimmo10, I have updated the latex support for TFM, and it renders as expected. Here is a sample -

![image](https://user-images.githubusercontent.com/53227127/124145286-490fcd80-daaa-11eb-8e93-371f87c890d2.png)

However, I am not well versed with the `pprint` parser. I tried to implement subscript in `pprint`, and the results were unsatisfactory. If you could help.

Reply:
I think in this case you need to define `_print_TransferFunctionMatrix` in _mathml.py_ file. Once you're done with that, one test can be added in _test_mathml.py_.

Reply:
> I think in this case you need to define `_print_TransferFunctionMatrix` in _mathml.py_ file. Once you're done with that, one test can be added in _test_mathml.py_.

I'll do that, but how will the MathML parser help us in getting the pretty-printed output?